### PR TITLE
i18n(fr): Update `reference/plugins.mdx` from #2549

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description
[i18n(fr): Update](https://github.com/withastro/starlight/commit/99c88f2ceb2a436a928a5048a9717e641cfe8f5f) [reference/plugins.mdx](https://github.com/withastro/starlight/commit/99c88f2ceb2a436a928a5048a9717e641cfe8f5f) [from](https://github.com/withastro/starlight/commit/99c88f2ceb2a436a928a5048a9717e641cfe8f5f) https://github.com/withastro/starlight/pull/2549

### CODE REVIEW

File: docs/src/content/docs/fr/resources/plugins.mdx [hunk 1]
```diff
@@ -93,6 +93,16 @@ Les [modules d'extension](/fr/reference/plugins/) peuvent personnaliser la confi
...
 	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-sidebar-topics"
+		title="starlight-sidebar-topics"
+		description="Divisez votre documentation en différentes sections, chacune ayant sa propre barre latérale."
+	/>
+	<LinkCard
+		href="https://github.com/trueberryless-org/starlight-sidebar-topics-dropdown"
+		title="starlight-sidebar-topics-dropdown"
+		description="Divisez votre page de documents en plusieurs sous-pages et passez de l'une à l'autre à l'aide d'un menu déroulant dans la barre latérale."
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/resources/plugins.mdx:104]:
```suggestion
		description="Divisez vos pages de documentation en plusieurs sous-pages et passez de l'une à l'autre à l'aide d'un menu déroulant dans la barre latérale."
```

I think "votre page de documents" may be a bit weird? I think the intent is to reference multiple pages of a documentation.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new translation 🌟 

Left a tiny comment to make it easier to understand the plugin behavior but otherwise, looks perfect to me 👍

File: docs/src/content/docs/fr/resources/plugins.mdx (hunk 1 above)
Comment (thomasbnt) [reply docs/src/content/docs/fr/resources/plugins.mdx:104]: Oh yeah it's better! Thanks!

**Comment by thomasbnt** [comment]
> Thanks for the new translation 🌟
> 
> Left a tiny comment to make it easier to understand the plugin behavior but otherwise, looks perfect to me 👍

Thanks for the review @HiDeoo !

**Review by HiDeoo — APPROVED** [review]
Thanks for the update :rocket:

**Pull request merged**